.circleci/config.yml

```diff
@@ -234,3 +249,22 @@ jobs:
       - publish_docker_images:
           repo: opentelemetry-collector-dev
           tag: ${CIRCLE_SHA1}
+
+  windows-test:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - restore_module_cache:
+          os: "windows"
+      - run:
+          name: Upgrade golang
+          shell: powershell.exe
+          command: choco upgrade golang --version=1.14.3
```

Comment:
Would be good if I can cache this operation since it takes a while. If anyone knows how, I would be happy to review a followup PR :)

Reply:
Doesn't sound like its possible from the CircleCI docs: "The Windows jobs are VM-based and provide complete build isolation. Each new job uses a clean environment that gets created just in time and gets destroyed once the job finishes running." - that pretty much rules out any kind of caching I would think 🙁

They'll probably improve support for Windows agents (or containers) in the future.

Reply:
A few of the tests are abnormally slow on Windows as well, I'll try to take a quick look at that if I get a chance.